README.md

```diff
@@ -1,3 +1,22 @@
+# <div align="center"> 🎉Taipy is taking part in Hacktoberfest 2023!🎉 </div>
```

Comment:
What is the plan when Hacktoberfest is over in a month from now?
Update the README again?
Do you really plan to ship 3.0 with THAT in it??

The code repository, in my opinion, is not a communication tool.
My 2 cents.

Reply:
Marine is in charge of this. She plans to update the README at the end of the hacktoberfest

Reply:
The README file is part of the product.
Not the communication on the product.
The product is built into something that we publish. Then gets visible for a long time (fortunately).
We will not create a 3.0.1 just because hacktoctoberfest is over. Makes very little sense.